client/core/core_test.go

```diff
@@ -2825,6 +2864,7 @@ func TestTradeTracking(t *testing.T) {
 	if len(proof.TakerSwap) == 0 {
 		t.Fatalf("swap not broadcast with confirmations")
 	}
+	<-rig.db.updateMatchChan // initsig update after init response
```

Comment:
One of a few new receives for the auth.InitSig and auth.Redeem sig that come after the requests.